openshift-install-powervs

```diff
@@ -1356,6 +1375,10 @@ function main {
       warn "Enabling forceful destruction option for terraform destroy command"
       FORCE_DESTROY=1
       ;;
+    "-force-run")
+      warn "Enabling forceful run option for executing utility."
```

Comment:
Change the message to "Enabling forceful run on unsupported Operating Systems"

Reply:
Resolved.